pkg/cmd/roachtest/test_runner.go

```diff
@@ -913,6 +917,47 @@ fi'`
 	}
 }
 
+// getGoCoverArtifacts retrieves the go coverage artifacts for the test.
+// If there's an error, oh well, don't do anything rash like fail a test
+// which already passed.
+func getGoCoverArtifacts(ctx context.Context, l *logger.Logger, c *clusterImpl, t test.Test) {
```

Comment:
Unless I'm missing something, sharing a function for fetching perf and coverage artifacts would be very simple so I don't see why not.